### ISSUE TITLE
Add option to disable context menu button

### ISSUE THREAD

**Issue by WildGenius** [languagetool-org/languagetool#1772]
Currently, there is a "Check spelling and grammar..." button when selecting text and right clicking and no way to remove it from the menu.

**Comment by danielnaber** [comment]
Are you referring to the browser add-on?

**Comment by WildGenius** [comment]
That's right, specifically the chrome version. Sorry, should've specified.

**Comment by tiff** [comment]
HI @WildGenius, just curious why you would want this: Is it annoying or is it hindering you in any way?

**Comment by WildGenius** [comment]
I like to keep my context menu as clean as possible and I am unlikely to use this particular button. It's a cosmetic issue, so no hurry.

**Comment by tiff** [comment]
Thanks for clarification
